projects/admin/src/app/circulation/patron/loan/loan.component.ts

```diff
@@ -327,7 +328,7 @@ export class LoanComponent implements OnInit, OnDestroy {
           errorMessage = err.error.message;
         }
         if (err.error.status === 403) {
-          let message = errorMessage || this._translateService.instant('Checkout is not allowed by circulation policy');
+          let message = this._translateService.instant(errorMessage) || this._translateService.instant('Checkout is not allowed by circulation policy');
```

Comment:
Make this string `Checkout is not allowed by circulation policy` into the file `manual_translations`